doc/content/source/bcs/MFEMVectorNormalIntegratedBC.md

```diff
@@ -0,0 +1,20 @@
+# MFEMVectorNormalIntegratedBC
+
+## Summary
+
+!syntax description /BCs/MFEMVectorNormalIntegratedBC
+
+## Overview
+
+Adds the boundary integrator for integrating the linear form
+
+!equation
+(\vec f \cdot \vec n, v)_{\partial\Omega} \,\,\, \forall v \in V
+
+where $v \in H^1$ and $\vec n$ is the outward facing unit normal vector on the surface.
```

Comment:
Do we want to use $\vec n$ or $\hat n$ throughout for unit normals?

Reply:
I prefer $\hat n$, as it's clearer that it's a unit normal vector rather than just any normal vector - I'll update this

Reply:
Updated, both here and in the source file

Reply:
Thanks for doing this Alex, sorry if I stepped on your toes when I changed the "surfaces" wording. I didn't completely understand if you wanted me to do it or not. :)